examples/meta/generator/parse.py

```diff
@@ -251,6 +253,10 @@ def p_enum(self, p):
         "enum : ENUMKEYWORD identifier DOT identifier"
         p[0] = {"Enum": [p[2], p[4]]}
 
+    def p_new(self, p):
```

Comment:
i think we should move this to a different PR (including a meta example for testing) to de-clutter the PR a bit?

Reply:
Yup, I'll do that. We probably need one for constructors, i.e. std::make_shared and one for functions that return raw pointers, i.e. std::shared_ptr